src/sentry/api/serializers/models/group_stream.py

```diff
@@ -156,6 +160,34 @@ def get_stats(
             return self.query_tsdb(item_list, query_params, user=user, **kwargs)
 
 
+# Serializer for External Issues Model
+# Maps an external issue to to additional integration information such as key or name
+class ExternalIssueSerializer(Serializer):
+    def get_attrs(self, item_list: List[ExternalIssue], user: User, **kwargs: Any):
+        result = {}
+        for ei in item_list:
```

Comment:
nit (and i know this is from the previous code), but small initial variable names like this aren't preferred, simply calling it `item` here would be okay, and is convention.